Sources/NIOPosix/HappyEyeballs.swift

```diff
@@ -70,6 +70,28 @@ public struct NIOConnectionError: Error {
     }
 }
 
+extension NIOConnectionError: CustomStringConvertible {
+    public var description: String {
+        if let dnsError = (dnsAError ?? dnsAAAAError) {
```

Comment:
nit: coding style in NIO is to use explicit `self`:

```suggestion
        if let dnsError = (self.dnsAError ?? self.dnsAAAAError) {
```

Reply:
Thanks - updated.